docs/content/toc.md

```diff
@@ -130,6 +130,7 @@ layout: toc
   * [Different Hadoop Versions](/docs/VERSION/operations/other-hadoop.html)
   * [High Availability](/docs/VERSION/operations/high-availability.html)
   * [Performance FAQ](/docs/VERSION/operations/performance-faq.html)
+  * [GUIs](/docs/VERSION/operations/guis.html)
```

Comment:
How about changing this to 'Management GUIs' here and in other places? This way people don't get it confused with data visualization UIs

Reply:
Changed to Management GUIs here and elsewhere